Mathlib/AlgebraicGeometry/AffineScheme.lean

```diff
@@ -276,6 +287,17 @@ theorem range_fromSpec :
 @[simp]
 theorem opensRange_fromSpec : Scheme.Hom.opensRange hU.fromSpec = U := Opens.ext (range_fromSpec hU)
 
+@[simp, reassoc]
```

Comment:
According to [this post](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/reassoc.20and.20simp/near/449310703) this should be `@[reassoc (attr := simp)]`, and there is an explanation there about how to check that this works.